templates/react/field-types/Timestamp/sample.hbs

```diff
@@ -0,0 +1 @@
+1674053334548000
```

Comment:
here's the value that gave me the date `55018-08-22, 12:09:08 a.m.` :)

Reply:
Fixed, using current date as sample: `Date.now() * 1000`